components/external-services-mock/pkg/webhook/webhook.go

```diff
@@ -0,0 +1,83 @@
+package webhook
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+type OKRequestData struct {
+	OK bool
+}
+
+type OperationResponseData struct {
+	Status string `json:"status"`
+}
+
+const (
+	OperationPath                     = "webhook/delete/operation"
+	DeletePath                        = "webhook/delete"
+	OperationResponseStatusOK         = "SUCCEEDED"
+	OperationResponseStatusINProgress = "IN_PROGRESS"
+)
+
+var isOk bool
```

Comment:
`isOk` is too general for me, I can't get any semantics for its meaning until I've looked around in the handlers. Maybe rename it to something like `deletionInvoked` or `hasDeletionStarted`, or something which has more meaning.